x/authz/simulation/operations.go

```diff
@@ -21,16 +21,16 @@ import (
 
 // authz message types
 const (
-	TypeMsgGrantAuthorization  = "/cosmos.authz.v1beta1.Msg/GrantAuthorization"
-	TypeMsgRevokeAuthorization = "/cosmos.authz.v1beta1.Msg/RevokeAuthorization"
-	TypeMsgExecDelegated       = "/cosmos.authz.v1beta1.Msg/ExecAuthorized"
+	TypeMsgGrantAuthorization  = "/cosmos.authz.v1beta1.Msg/Grant"
+	TypeMsgRevokeAuthorization = "/cosmos.authz.v1beta1.Msg/Revoke"
+	TypeMsgExecDelegated       = "/cosmos.authz.v1beta1.Msg/Exec"
```

Comment:
These will need to be changed with #9139

Reply:
Do we want to do it now (here), or in #9139?

Reply:
I actually did it in #9139 already.